### DISCUSSION TITLE
Q&A / 问与答

### DISCUSSION THREAD

**Discussion by Xiaokang2022Vice** [Xiaokang2022/maliang#39] [Q&A] [answered]
**Q1:** *How do I ask a question about the code for the project?*  
怎么提出关于该项目在代码上的问题？

**Q2:** *How do I share my experience and insights about the project?*  
怎么分享自己关于该项目的经验和心得？

**Q3:** *How can I make some suggestions for the project?*  
如何对该项目提出一些建议？

**Q4:** *There is a problem with the documentation site, should I open an issue in the main repository or in the documentation repository?*  
文档站存在问题，应该在主仓库发起 Issue 还是在文档仓库？

**Q5:** *How do I delete a comment that I wrote myself in the comment section at the bottom of the official website and documentation site?*  
如何删除在官方网站和文档网站底部评论区中自己写的评论？

**Q6:** *How can I get my issues resolved as soon as possible?*  
如何让自己提出的 Issue 能被尽快解决？

**Q7:** *Want to contribute code to the project, but feel like the requirements for [contributing guidelines](https://github.com/Xiaokang2022/tkintertools/blob/main/CONTRIBUTING.md) are too complicated, can I contribute code without meeting those requirements?*  
想给该项目贡献代码，但感觉[贡献指南](https://github.com/Xiaokang2022/tkintertools/blob/main/CONTRIBUTING.md)的要求太复杂了，可以在不满足那些要求的情况下贡献代码吗？

**Q8:** *How to urge the author to update quickly?*  
如何催更？

**Q9:** *What's going on with this account?*  
这个账号是怎么回事？

**Comment by Xiaokang2022** [comment, marked as answer]
**A1:** *You can go to the [official website](https://xiaokang2022.github.io/tkintertools/) of the project first, check out the tutorials and documentation, and if it still doesn't solve your problem, you can try to ask your question in the [discussion forum](https://github.com/Xiaokang2022/tkintertools/discussions/categories/q-a), and of course, if you want, you can also ask your question in the comment section at the bottom of the documentation station.*  
你可以先去该项目[官方网站](https://xiaokang2022.github.io/tkintertools/)，查看教程和文档，若仍不能解决你的问题，可以尝试在[讨论区](https://github.com/Xiaokang2022/tkintertools/discussions/categories/q-a)提出的你的问题，当然，如果你愿意，你也可以在文档站底部的评论区中提出你的问题。

**A2:** ~~*You can start a discussion in the ["Ideas"](https://github.com/Xiaokang2022/tkintertools/discussions/categories/ideas) category in the project's discussion board and write your thoughts on it.*  
你可以在该项目讨论区的分类[“想法”](https://github.com/Xiaokang2022/tkintertools/discussions/categories/ideas)中发起一个讨论，并将自己的心得写进去。~~  
见：https://xiaokang2022.github.io/tkintertools/blog/

**A3:** *Make your suggestion by initiating an [issue](https://github.com/Xiaokang2022/tkintertools/issues/new/choose).*  
通过发起一个[议题](https://github.com/Xiaokang2022/tkintertools/issues/new/choose)来提出您的建议。

**A4:** *Anywhere.*  
无论哪里都可以。

**A5:** *The comment mechanism is actually implemented by the open source project [giscus](https://github.com/giscus/giscus), it is actually a mapping of specific categories (Here is [GitHub Pages](https://github.com/Xiaokang2022/tkintertools/discussions/categories/github-pages)) of the discussion board, you can find the relevant link in the comment section at the bottom of the page, click on it to jump to the source discussion board, where you can delete your own comments.*  
评论机制实际是由开源项目 [giscus](https://github.com/giscus/giscus) 实现的，它实际是讨论区特定分类（这里是 [GitHub Pages](https://github.com/Xiaokang2022/tkintertools/discussions/categories/github-pages)）的映射，你可以在页面底部评论区中找到相关链接，点进去既可以跳转到源讨论区，在那里可以删除你自己的评论。

**A6:** *The order and priority of issues to be resolved is determined by the maintainers, but if you want your issues to have a higher priority, you can try to [sponsor the project](https://xiaokang2022.github.io/tkintertools/Sponsor/) (be sure to state your requirements at the time of sponsorship) so that your issues will be resolved first.*  
解决议题顺序及优先级是由维护人员决定的，如果你想让你的议题能有较高的优先级，可以尝试[赞助该项目](https://xiaokang2022.github.io/tkintertools/Sponsor/)（务必在赞助时说明的你的要求），这样你的议题会被优先解决。【人话：得加钱】

**A7:** *Depending on the situation, if your contribution implements a very useful feature or solves an important problem, then it is acceptable if it does not meet any of the requirements.*  
依情况而定，如果你贡献的代码实现了非常有用的功能，或者解决了很重要的问题，那么就算不满足哪些要求也是可以被接受的。

**A8:** *#?$@/&,6...*
歪比巴卜……

**A9:** *This account is my annother account and is for testing and auxiliary purposes only.*  
此账号是我小号，仅用于测试和辅助用途。
